modules/rostests/apitests/ntdll/UserModeException.c

```diff
@@ -0,0 +1,215 @@
+/*
+* PROJECT:     ReactOS api tests
+* LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+* FILE:        modules/rostests/apitests/ntdll/UserModeExceptions.c
+* PURPOSE:     Tests for user mode exceptions
+* PROGRAMMERS: Copyright 2021 Timo Kreuzer (timo.kreuzer@reactos.org)
+*/
```

Comment:
```suggestion
/*
 * PROJECT:     ReactOS API tests
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Tests for user mode exceptions
 * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
 */
```

Reply:
Year should certainly be 2022, not 2021.

Reply:
I guess this patch is from 2021, but it's up to Timo whether to update the year